core/templates/dev/head/pages/exploration_editor/history_tab/HistoryServices.js

```diff
@@ -205,9 +205,9 @@ oppia.factory('compareVersionsService', [
           var lca = versionsTreeService.findLCA(v1, v2);
 
           var v1States = StatesObjectFactory.createFromBackendDict(
-            v1StatesDict);
+            v1StatesDict).getStates();
```

Comment:
Oh wait nevermind I think I get what you're saying. Just have a toDict method that has one "States" property?